examples/stm32/nucleo-h563zi-make-baremetal-builtin/main.c

```diff
@@ -77,7 +77,7 @@ int main(void) {
   }
 
   MG_INFO(("Initialising application..."));
-  web_init(&mgr);
+  //web_init(&mgr);
```

Comment:
Is there a reason for this change?

Reply:
Oh, good catch. I did that for testing and forgot to re-enable. Let me cleanup..